### PR TITLE
config: update config template

### PR DESCRIPTION
**Pull request by jackysp**
1. Fix typos.
2. Add upper limit and some default info.
3. Make template values consistent with default.
@coocood @tiancaiamao PTAL

### CODE REVIEW

**Comment by zz-jason** [comment]
@jackysp plz fix ut

File: config/config.toml.example [hunk 1]
```diff
@@ -13,10 +13,10 @@ store = "mocktikv"
 path = "/tmp/tidb"
 
 # The socket file to use for connection.
-#socket = ""
+socket = ""
```
Comment (shenli) [inline config/config.toml.example]: Why uncomment it?

Comment (jackysp) [reply config/config.toml.example]: Make it consistent with other settings in the config example file.

**Comment by jackysp** [comment]
PTAL @coocood

**Comment by coocood** [comment]
LGTM

**Pull request merged**